src/__tests__/resources/assets.unit.spec.ts

```diff
@@ -198,4 +199,49 @@ describe('Asset unit test', () => {
       });
     });
   });
+
+  describe('shallow copy + JSON.stringify()', () => {
+    const items = [
+      {
+        id: 1,
+      },
+      {
+        id: 2,
+      },
+    ];
+
+    test('assets.list().autoPagingToArray()', async () => {
+      nock(mockBaseUrl)
+        .post(new RegExp('/assets/list'), {})
+        .once()
+        .reply(200, { items });
+
+      const assets = await client.assets.list().autoPagingToArray({ limit: 2 });
+      expect(JSON.stringify(assets)).toBeTruthy();
+      const shallowCopyItems: TypeAsset[] = [...assets, assets[0]];
+      expect(JSON.stringify(shallowCopyItems)).toBeTruthy();
+      const destructedAsset: TypeAsset = { ...assets[1] };
+      expect(destructedAsset.id).toBeDefined();
+      expect(() => JSON.stringify(destructedAsset)).toThrowError();
+      const destructedJsonAsset: TypeAsset = { ...assets[1].toJSON() };
+      expect(JSON.stringify(destructedJsonAsset)).toBeTruthy();
+    });
+
+    test('assets.list()', async () => {
```

Comment:
I don't see the need for this test? Duplicate of the one above? If we are using autoPaginate or not shouldn't matter or no?